resources/qml/Settings/SettingTextField.qml

```diff
@@ -223,7 +223,7 @@ SettingItem
 
                 cursorShape: Qt.IBeamCursor
 
-                onPressed: {
+                onPressed:(mouse)=> {
```

Comment:
I never saw this QML syntax :smile: what does it do ?

Reply:
Oh yes indeed, because `mouse` is used implicitely inside the function, so that makes it explicit. Good to know :smiley: :+1: